modules/apps/content-dashboard/content-dashboard-blogs-impl/src/main/java/com/liferay/content/dashboard/blogs/internal/item/type/BlogsEntryContentDashboardItemSubtype.java

```diff
@@ -47,7 +48,10 @@ public String getFullLabel(Locale locale) {
 
 	@Override
 	public InfoItemReference getInfoItemReference() {
-		return new InfoItemReference(BlogsEntry.class.getName(), 0);
+		return new InfoItemReference(
+			BlogsEntry.class.getName(),
```

Comment:
@jkappler !!

Maybe it is better if we support ClassNameClassPKInfoItemIdentifier and no ClassNameClassPKInfoItemIdentifier InfoIdentifier?


Thanks!!

Reply:
Then we would need to add support for all types of infoItemIdentifiers right @cgoncas?

Reply:
Something like that, because as this is an API, maybe we will face this problem with other kind of InfoItems :(